backend/firebase.config

```diff
@@ -0,0 +1,10 @@
+const FIREBASE_CONFIGS = {
+  apiKey: "AIzaSyDWxBCf-A_uCeRpzhwyrF8HaVqHLUzEu_o",
+  authDomain: "decision-io.firebaseapp.com",
+  databaseURL: "https://decision-io.firebaseio.com",
```

Comment:
The config files should not be committed. The .gitignore was updated in a another PR to also ignore config files without the .js at the end

Reply:
I will change this later